lib/Page.js

```diff
@@ -694,6 +694,43 @@ class Page extends EventEmitter {
   }
 
   /**
+   * @param {string} selector
+   * @param {string|string[]} value
+   */
+  async select(selector, value) {
+    const handle = await this.$(selector);
+    console.assert(handle, 'No node found for selector: ' + selector);
+
+    await this.evaluate((element, value) => {
+      if (element.nodeName.toLowerCase() !== 'select')
+        return false;
+
+      if (Array.isArray(value) !== true)
+        value = [value];
```

Comment:
nit: the `value` is always an array; let's call it `values` to make it clear

Reply:
This was how I first wrote, but since it can also be provided to the method as a string, I thought `value` was more appropriate in either case - but I'll change it. :+1: